.github/workflows/github-actions.yml

```
@@ -0,0 +1,21 @@
+name: 'Check test, code style and linter'
+'on':
+  - push
+  - pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - name: Install and configure Poetry
+        uses: snok/install-poetry@v1
+      - name: Install dependencies
+        run: |
+          cd Python/crossfire
+          poetry install
+      - name: Run tests
+        run: poetry run pytest
```

Pelo jeito precisamos do `cd Python/crossfire` no comando dos testes tb : )

```suggestion
        run: |
          cd Python/crossfire
          poetry run pytest
```

---

Volto a isso quando tivermos entendido o erro....